core/tests/puppeteer-acceptance-tests/utilities/user/curriculum-admin.ts

```diff
@@ -807,6 +807,60 @@ export class CurriculumAdmin extends BaseUser {
   async openExplorationControlDropdown(): Promise<void> {
     await this.clickOn(explorationControlsSettingsDropdown);
   }
+
+  async unpublishTopic(topicName: string): Promise<void> {
+    await this.openTopicEditor(topicName);
+    await this.clickOn(' Unpublish Topic ');
+  }
+
+  async deleteTopic(topicName: string): Promise<void> {
+    await this.goto(testConstants.URLs.TopicAndSkillsDashboard);
+    await this.clickOn('.e2e-test-topic-edit-box');
+    await this.clickOn('Delete');
+  }
+
+  async deleteSkill(skillName: string): Promise<void> {
+    await this.goto(testConstants.URLs.TopicAndSkillsDashboard);
+    await this.clickOn('.e2e-test-skills-tab');
+
+    await this.clickOn('.e2e-test-skill-edit-box');
+    await this.clickOn('Delete');
+  }
+
+  async expectTopicNotInTopicsAndSkillDashboard(
+    topicName: string
+  ): Promise<void> {
+    await this.goto(testConstants.URLs.TopicAndSkillsDashboard);
+
+    const isTopicPresent = await this.isTextPresentOnPage(topicName);
+    if (isTopicPresent) {
+      throw new Error(
+        `Expected topic "${topicName}" not to be present on the page, but it was.`
```

Comment:
Maybe rewrite this sentence to "Topic ${topicName} is present on the page unexpectedly"

Reply:
Done.